src/libraries/Microsoft.Extensions.Logging/src/Microsoft.Extensions.Logging.csproj

```
@@ -23,7 +23,7 @@
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.DependencyInjection.Abstractions\src\Microsoft.Extensions.DependencyInjection.Abstractions.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Logging.Abstractions\src\Microsoft.Extensions.Logging.Abstractions.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Options\src\Microsoft.Extensions.Options.csproj" />
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.DiagnosticSource\src\System.Diagnostics.DiagnosticSource.csproj" />
+    <ProjectReference Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)' OR '$(TargetFramework)' != '$(NetCoreAppMinimum)'" Include="$(LibrariesProjectRoot)System.Diagnostics.DiagnosticSource\src\System.Diagnostics.DiagnosticSource.csproj" /> 
```

```suggestion
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
    <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.DiagnosticSource\src\System.Diagnostics.DiagnosticSource.csproj" /> 
```

1. It is more readable to put the condition on the ItemGroup.
2. Can simplify the condition to just use TargetFrameworkIdentifier
3. I believe the `OR` is incorrect logic - It should have been `AND`.

---

> I believe the OR is incorrect logic - It should have been AND.

That is right. I missed this part. Thanks for the catch.